Formula/w/wxpython.rb

```diff
@@ -29,6 +29,12 @@ class Wxpython < Formula
     depends_on "gtk+3"
   end
 
+  # Backport increase of SIP ABI to fix build
+  patch do
+    url "https://github.com/wxWidgets/Phoenix/commit/de9aa4be5bb49adf82991c7582ea3c42ed505bf7.patch?full_index=1"
+    sha256 "bf752fa850459d963cf8a7678dd0463934888d9867a9ac80d58ca51d19cb9f93"
+  end
+
```

Comment:
maybe rebottle?

Reply:
I think it is only a build issue as we upgraded our version of `sip`.

Existing bottles should still be fine and are less likely to have unexpected results if built with similar `sip` to upstream.

Reply:
sounds good 👍